src/extensions/pxblue-extensions.ts

```diff
@@ -119,6 +120,58 @@ module.exports = (toolbox: GluegunToolbox): void => {
             });
         }
 
+        // Map the template selection to template src
+        let templateSrc = '';
+        switch (template.toLocaleLowerCase()) {
+            case 'basic routing':
```

Comment:
Support "routing" (can come from command line flag) or "basic routing" (comes from the interactive prompt)
```suggestion
            case 'basic routing':
            case 'routing':
```